v8-compile-cache.js

```diff
@@ -304,7 +304,13 @@ function getCacheDir() {
     : typeof process.versions.chakracore === 'string'
       ? 'chakracore-' + process.versions.chakracore
       : 'node-' + process.version;
-  const cacheDir = path.join(os.tmpdir(), dirname, version);
+  const cacheDir = path.join(
+    process.env && process.env.V8_COMPILE_CACHE_BASE_DIR
```

Comment:
process.env is always defined

Reply:
Not in those tests :) try it...

Reply:
hmm, is that to cover a specific node version where this was not true? Otherwise I'd say that's an incorrectly mocked test env.

Reply:
@hulkish Here are the test failures from travis. It has to do with the 'vm' used in the tests I assume: https://travis-ci.org/zertosh/v8-compile-cache/builds/509071381

Reply:
@simlu ok, fair enough - although, it does seem like a mistake in the test envs...i agree it shouldn't necessarily block this pr, unless maintainer decides.